eng/pipeline/stages/run-stage.yml

```diff
@@ -206,6 +206,11 @@ stages:
                   # The Chocolatey shims are located in a single folder in PATH, so we can't change PATH to exclude it.
                   # Upstream Windows builders don't have SWIG installed, so this makes coverage even.
                   RemovePathBinary 'swig'
+                  
+                  Write-Host "Removing 'patch' to avoid parts of cmd/go TestScript/mod_tidy_diff on Windows."
```

Comment:
Do we need to report this issue upstream in the poweshell/Windows patch tool?

Reply:
IMO: no. This patch.exe seems to come from Strawberry Perl, looking at CI logs: `Removing C:\Strawberry\c\bin\patch.exe`. (I couldn't tell until seeing it in CI, and didn't repro locally.) I anticipate the Strawberry Perl team telling us this binary isn't intended to implement the same set of features and match the Linuxy output/result 1:1.

Before going any further, I'd want to grab the Strawberry Perl version of patch.exe and see the results for myself so I know what's actually going on. `exit status 57005` is pretty odd. We might even need to change CI to export its `patch.exe` specifically because it might be hard to reproduce how it's set up.

I think there is potential for some value if we looked into it, but I don't think it's worth the effort currently.